includes/Story_Renderer/HTML.php

```diff
@@ -352,10 +352,14 @@ protected function add_publisher_logo() {
 			return;
 		}
 
-		$publisher_logo = $story_element->getAttribute( 'publisher-logo-src' );
+		$publisher_logo_src = $story_element->getAttribute( 'publisher-logo-src' );
+		if ( empty( $publisher_logo_src ) || $publisher_logo_src === $this->get_publisher_logo_placeholder() ) {
+			$story_element->removeAttribute( 'publisher-logo-src' );
```

Comment:
If there is no publisher logo, need to do `$html->removeAttribute( 'amp' );` like `add_poster_images` does below

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/4787/commits/c45f2726751bc701a54b7c268e9b3e66947a6a1d